modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/modal/Modal.js

```diff
@@ -331,6 +331,9 @@ const openConfirmModal = ({message, onConfirm, title}) => {
 	else if (confirm(message)) {
 		onConfirm(true);
 	}
+	else {
+		onConfirm(false);
+	}
```

Comment:
It may be simpler to do this

```suggestion
	else {
		onConfirm(confirm(message));
	}
```

I think? 🤔

Reply:
sounds good! thank you, added it in